docs/deployment.md

```diff
@@ -1,68 +1,67 @@
 ---
-description: Deploy your Next.js app to production with Vercel and other hosting options.
+description: Learn how to deploy your Next.js app to production either managed or self-hosted.
 ---
 
 # Deployment
 
-## Vercel (Recommended)
+Congratulations, you are ready to deploy your Next.js application to production. This document will show how to deploy either managed or self-hosted using the [Next.js Build API](#nextjs-build-api).
 
-The easiest way to deploy Next.js to production is to use the **[Vercel platform](https://vercel.com)** from the creators of Next.js. [Vercel](https://vercel.com) is a cloud platform for static sites, hybrid apps, and Serverless Functions.
+## Next.js Build API
 
-### Getting started
+`next build` generates an optimized version of your application for production. This standard output includes:
 
-If you haven’t already done so, push your Next.js app to a Git provider of your choice: [GitHub](https://github.com/), [GitLab](https://gitlab.com/), or [BitBucket](https://bitbucket.org/). Your repository can be private or public.
+- HTML files for pages using `getStaticProps` or [Automatic Static Optimization](/docs/advanced-features/automatic-static-optimization.md)
```

Comment:
Id like to see us drop the period on list items, per the style guide. But its tough to enforce. Reason being that people forget to add them sometimes, like on this one. I also think it looks a little cleaner. 

This file includes lists with some periods, and a bunch of other lists without them 😅 I'll let you decide what to do here, but consistency is obv key. It looks shabby when some lists have them and others dont.

Reply:
How do you handle bullets with multiple sentences then? e.g. the first sentence has a period, but the next one doesn't

Reply:
Went ahead and tried it out with no period here 👍